### PR TITLE
Update ONNX Runtime Web to 1.19 dev for Image Classification demos

### PR DESCRIPTION
**Pull request by ibelem**
The CL [#5494397 Remove support for passing MLActivations for op fusion](https://chromium-review.googlesource.com/c/chromium/src/+/5494397) landed in Chromium but the [PR #20635 remove activation fusion](https://github.com/microsoft/onnxruntime/pull/20635) was not included in ONNX Runtime Web 1.18.0 and it was unable to be cherry picked into 1.18.1.

It causes the [ResNet50 fp16 demo](https://microsoft.github.io/webnn-developer-preview/demos/image-classification/?provider=webnn&devicetype=gpu&model=resnet-50&run=1) failure on Canary and Beta of Chrome.

| Chrome Browser | Result| Note | 
|----|----|----|
| Canary 128.0.6554.0 | FAIL | Top 1 label - tench, Tinca tinca	NaN |
| Beta 127.0.6533.17 | FAIL  | Top 1 label - tench, Tinca tinca	NaN |
| Stable 126.0.6478.115 | PASS| Top 1 label - tiger, Panthera tigris	0.9231255054473877|

### Changes

1. This PR updated the ONNX Runtime Web to latest 1.19 dev and updated the Transformer.js built with 1.19 dev. 
2. Replaced "vite" by "http-server" for local test environment only since the Vite JS injection for Hot Module Replacement (HMR) support breaks the functionality of ORT Web 1.19, unless to change the code/folder structure. It will not impact the hosted site e.g. microsoft.github.io or forked github.io pages.

@fdwr @Adele101 PTAL

### CODE REVIEW

File: demos/image-classification/index.js [hunk 1]
```diff
@@ -466,6 +467,24 @@ const changeImage = async () => {
   await main();
 };
 
+const setupORT = async () => {
+  const ortversion = document.querySelector("#ortversion");
+  removeElement("onnxruntime-web");
+  let ortVersion = "1.19.0-dev.20240621-69d522f4e9";
+  let ortLink = "";
+  if (ortVersion && ortVersion.length > 4) {
```
Comment (fdwr) [inline demos/image-classification/index.js:475]: What is the meaning of `> 4`? Does that mean an ORT release of `1.20` (4 characters) would pick the older local build, even though it's newer than `1.19.0`?

**Review by fdwr — COMMENTED** [review]
Some comments Belem. Thanks for the updates.

File: demos/image-classification/index.js [hunk 2]
```diff
@@ -466,6 +467,24 @@ const changeImage = async () => {
...
+    await loadScript(
+      "onnxruntime-web",
+      `https://cdn.jsdelivr.net/npm/onnxruntime-web@${ortVersion}/dist/ort.all.min.js`
+    );
+    ortLink = `https://www.npmjs.com/package/onnxruntime-web/v/${ortVersion}`;
+    ortversion.innerHTML = `ONNX Runtime Web: <a href="${ortLink}">${ortVersion}</a>`;
+  } else {
+    await loadScript("onnxruntime-web", "../dist/ort.all.min.js");
```
[the remark this answers is not in the document]
Comment (ibelem) [reply demos/image-classification/index.js:483]: @fdwr You are right, I will remove `else` clause for Image Classification.

File: demos/image-classification/index.js (hunk 1 above)
Comment (ibelem) [reply demos/image-classification/index.js:475]: This function was used in Web AI Benchmark to check ORT Web builds from public, and even internal built by @Honry. I will make it simpler for Image Classification.

**Comment by ibelem** [comment]
@fdwr Thanks for the review comments, and fixed these issues. PTAL.

**Review by fdwr — APPROVED** [review]
LGTM Sir.

**Pull request merged**